rest-api-spec/src/main/resources/rest-api-spec/test/indices.sort/10_basic.yml

```diff
@@ -11,7 +11,7 @@
         body:
           settings:
             number_of_shards: 1
-            number_of_replicas: 0
+            number_of_replicas: 1
```

Comment:
Not related to this PR but this will test the index sort when replicas are on different nodes.